Updated images in the team page

Suhanigarg01 opened the pull request.

amaan-aly246 reviewed (changes requested):
Do the following changes : 
- Crop Garv's photo
- Add Amaan and Samridhi's photos
- Share SS after the changes

Suhanigarg01 commented:
updated the changes and added all the new photos as well in the team page
<img width="1440" alt="Screenshot 2025-05-22 at 12 43 36 AM" src="https://github.com/user-attachments/assets/1fbfd50d-a916-469c-8a0b-3c32b850aea0" />
